server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -361,6 +367,15 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 			this.checkpointService,
 		);
 
+		// Emit api counter every certain interval.
+		setInterval(() => {
```

Comment:
Add an `if (this.deliCheckpointMetricInterval > 0) { }` around this setInterval line.
That will allow disabling these api counters when they aren't needed

Reply:
Thanks for the review. Added here.